json-rpc/spec.json

```diff
@@ -1716,6 +1716,11 @@
 									"maxPriorityFeePerGas"
 								],
 								"properties": {
+									"gasPrice": {
+										"title": "transactionGasPrice",
+										"type": "string",
+										"description": "The effective gas price paid by the sender in Wei. For transactions not yet mined, this value should be set equal to the max fee cap per gas. This field is DEPRECATED, please transition to using effectiveGasPrice in the Receipt object going forward."
```

Comment:
We should include a clear end of life (EOL) policy here.  I suggest something like "This field will no longer be returned by clients sometime after 2022-01-01."

Reply:
I'd change that to _may_ no longer be returned. And in practice I would probably recommend returning it indefinitely, since this will be a breaking change.